Source/EasyNetQ/Producer/DefaultRpc.cs

```diff
@@ -141,7 +139,7 @@ private void OnConnectionCreated(ConnectionCreatedEvent @event)
 
         protected void DeRegisterResponseActions(Guid correlationId)
         {
-            responseActions.TryRemove(correlationId, out _);
+            responseActions.Remove(correlationId);
```

Comment:
Strange, I do not see this method in ConcurrentDictionary API.

Reply:
Sure, it's an extension.